src/renderer/components/ui/QuitApp/QuitAppDialog.test.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+import { renderComponent } from '../../../testUtils/renderComponent'
+import { QuitAppDialog } from './QuitAppDialog'
+
+describe('QuitAppDialog', () => {
+  it('renders component', () => {
+    const result = renderComponent(
+      <QuitAppDialog open handleClose={jest.fn()} handleQuit={jest.fn()} />
+    );
+
+    expect(result).toMatchSnapshot()
```

Comment:
Looks great, but I think this should actually be

expect(result.toJSON()).toMatchSnapshot()

Reply:
![Uploading Zrzut ekranu 2021-09-29 o 08.11.39.png…]()